service/go.mod

```diff
@@ -179,3 +179,5 @@ require (
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20250603155806-513f23925822 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
+
+replace github.com/opentdf/platform/protocol/go => ../protocol/go
```

Comment:
Can you please remove this?

Reply:
I've removed it, but the "Friendly Reminders" check will fail.

Reply:
Yes, `protocol/go` and `service` are separate go modules, so to add the RPCs to the service and to add the obligation service client to the SDK, you'll need to:
1. merge the proto gencode
2. release `protocol/go`
3. bump `protocol/go` in the `sdk` module
4. release the `sdk` module
5. bump `protocol/go` and `sdk` in `service`

Since you're only adding RPCs to `service` that depend on unreleased proto types, I think you can skip the SDK-related bumps but would need a release of `protocol/go` for a tidy go mod in CI. The `go.work` resolves all this for you locally.